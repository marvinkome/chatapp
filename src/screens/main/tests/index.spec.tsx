import React from 'react';
import { shallow } from 'enzyme';

import MainScreen from '../index';

describe('main screen tests', () => {
    it('header should with 3 tabs', () => {
        const app = shallow(<MainScreen />);
        expect(app).toMatchSnapshot();
    });
});
