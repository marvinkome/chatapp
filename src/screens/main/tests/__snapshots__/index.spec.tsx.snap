// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main screen tests header should with 3 tabs 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Main />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "centerComponent": Object {
        "style": Object {
          "color": "#fff",
        },
        "text": "MY TITLE",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "centerComponent": Object {
          "style": Object {
            "color": "#fff",
          },
          "text": "MY TITLE",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
